queue_pause_opts.go

```diff
@@ -0,0 +1,4 @@
+package river
+
+// QueuePauseOpts are optional settings for pausing or unpausing a queue.
```

Comment:
I've been struggling with this myself just writing comments here, but for purposes of docs we should try to consolidate on one term. Either "resume" or "unpause", but try to stay consistent.